deliver/lib/deliver/upload_metadata.rb

```diff
@@ -451,10 +459,43 @@ def retry_if_nil(message, tries: 5, wait_time: 10)
       end
     end
 
-    # Finding languages to enable
-    def verify_available_info_languages!(options, app, languages)
-      app_info = fetch_edit_app_info(app)
+    # Checking if the metadata to update includes localised App Info
+    def updating_localised_app_info?(options, app, app_info)
+      app_info ||= fetch_live_app_info(app)
+      unless app_info
+        UI.important("Can't find edit or live App info. Skipping upload.")
+        return false
+      end
+      localizations = app_info.get_app_info_localizations
 
+      LOCALISED_APP_VALUES.keys.each do |key|
+        current = options[key]
+        next unless current
+
+        unless current.kind_of?(Hash)
+          UI.error("Error with provided '#{key}'. Must be a hash, the key being the language.")
+          next
+        end
+
+        current.each do |language, value|
+          strip_value = value.to_s.strip
+          next if strip_value.empty?
+
+          app_info_locale = localizations.find { |l| l.locale == language }
+          next if app_info_locale.nil?
+
+          current_value = app_info_locale.public_send(key.to_sym)
+
+          return true if current_value != strip_value
+        end
+      end
+
+      UI.message('No changes to localised App Info detected. Skipping upload.')
```

Comment:
💄  Same type of nitpicking about US vs UK spelling 🙃 
```suggestion
      UI.message('No changes to localized App Info detected. Skipping upload.')
```